packages/slice-machine/tests/mocks.test.js

```diff
@@ -1,16 +1,315 @@
-import * as Widgets from "@lib/models/common/widgets";
-import { handleFields } from "@lib/mock/misc/handlers";
-import { model, createExpector } from "./__mockData__/models";
+import * as Widgets from "../lib/models/common/widgets";
+import { handleFields } from "../lib/mock/misc/handlers";
 
-const mockFields = handleFields(Widgets);
+const toHaveProps = (obj, props) =>
+  props.forEach((prop) => expect(obj).toHaveProperty(prop));
 
-test("Can create mocks", () => {
-  const primary = Object.entries(model.variations[0].primary);
-  const mockEntries = Object.entries(mockFields(primary));
-  expect(mockEntries.length).toBe(primary.length);
+describe("handleFields", () => {
+  const mockFields = handleFields(Widgets);
+  test("can create a text mock", () => {
+    const textField = [
+      [
+        "text",
+        {
+          config: {
+            label: "",
+            placeholder: "",
+            allowTargetBlank: true,
+            single: "paragraph",
+          },
+          type: "StructuredText",
+        },
+      ],
+    ];
 
-  const expector = createExpector(expect);
-  mockEntries.forEach(([key, value]) =>
-    expector[key] ? expector[key](value) : null
-  );
+    const textFieldMocked = Object.entries(mockFields(textField))[0];
+    const [key, value] = textFieldMocked;
+    expect(Array.isArray(value)).toBe(true);
```

Comment:
I think to not have all of these, I would put the mock of the field in a mock file and then test the equality of it.
See if the function mock field for a given field gives the same result.

Reply:
These two comments apply for all the fields i think.

Reply:
Yeah I use now match object with expect inside it, it makes clearer the object that should be returned :)